occult/lib/task/firebase_login.dart

```diff
@@ -1,5 +1,10 @@
 import 'package:occult/util/tasks.dart';
+import 'package:universal_io/io.dart';
 
 class TFBLogin extends TRunInteractive {
-  TFBLogin() : super("firebase", const ["login"]);
+  TFBLogin()
```

Comment:
TFBLogin calls a process TRun task or process utility. It should handle the command switch internally same as everything